4_params_munge/src/munge_for_lakeattributes.R

```diff
@@ -0,0 +1,112 @@
+munge_area <- function(out_ind, area_ind){
+
+  #format for lakeattributes, which is column site_id and area_m2
+  area_dat <- readRDS(sc_retrieve(area_ind)) %>%
+    dplyr::select(-lake_name) %>%
+    filter(!is.na(area_m2))
+
+  data_file <- scipiper::as_data_file(out_ind)
+  saveRDS(area_dat, data_file)
+  gd_put(out_ind, data_file)
+}
+
+munge_depth <- function(out_ind, lagos_depth_ind, lagos_crosswalk, mglp_depth_ind){
+
+  lagos_depth_dat <- read.csv(sc_retrieve(lagos_depth_ind))
+  lagos_cross <- readRDS(sc_retrieve(lagos_crosswalk))
+  mglp_depth_dat <- readRDS(sc_retrieve(mglp_depth_ind))
+
+  # bring depth data together and format for lakeattributes
+  # columns are site_id, source, zmax_m
+  lagos_clean <- lagos_depth_dat %>%
+    mutate(LAGOS_ID = paste0('lagos_', lagoslakeid),
+           source = 'lagos') %>%
+    right_join(lagos_cross) %>%
+    dplyr::select(site_id, source, zmax_m = zmaxobs)
+
+  mglp_clean <- rename(mglp_depth_dat, zmax_m = z_max) %>%
+    mutate(source = 'mglp')
+
+  depth_dat <- bind_rows(lagos_clean, mglp_clean) %>%
+    distinct()
+
+  data_file <- scipiper::as_data_file(out_ind)
+  saveRDS(depth_dat, data_file)
+  gd_put(out_ind, data_file)
+}
+
+munge_secchi <- function(out_ind, lagos_secchi_ind, lagos_crosswalk){
+
+  lagos_secchi_dat <- readRDS(sc_retrieve(lagos_secchi_ind))
+  lagos_cross <- readRDS(sc_retrieve(lagos_crosswalk))
+
+  # format for lakeattributes
+  # columns are site_id, year, date, secchi_m, source, datasource
+
+  secchi_clean <- lagos_secchi_dat %>%
+    rename(LAGOS_ID = site_id) %>%
+    left_join(lagos_cross) %>%
+    filter(!is.na(site_id)) %>%
+    mutate(year = lubridate::year(date),
+           source = 'in-situ') %>%
+    dplyr::select(site_id, year, date, secchi_m = secchi, source, datasource = mon_group)
+
+
+
+  data_file <- scipiper::as_data_file(out_ind)
+  saveRDS(secchi_clean, data_file)
+  gd_put(out_ind, data_file)
+}
+
+munge_location <- function(out_ind, mglp_zip_ind, mglp_gdb_file, states){
+  # columns for location are site_id, lon, lat
+
+  zip_file <- scipiper::sc_retrieve(mglp_zip_ind)
+
+  shp.path <- tempdir()
+  unzip(zip_file, exdir = shp.path)
+
+  shp <- sf::st_read(file.path(shp.path, mglp_gdb_file), layer = 'MGLP_LAKES')
+
+  centroid <- shp %>%
+    filter(ASSESS == 'Y', STATE %in% states) %>%
+    mutate(centroid = st_centroid(SHAPE)) %>%
+    mutate(centroid = st_transform(centroid, crs = 4326))
```

Comment:
Again could make the `crs` a configuration

Reply:
I like this idea. I'm going to make an issue for me to incorporate it in the next PR (so it does not trigger a rebuild right now).